cli/Cargo.toml

```diff
@@ -47,3 +48,10 @@ ita-stf = { path = "../app-libs/stf" }
 itc-rpc-client = { path = "../core/rpc-client" }
 itp-time-utils = { path = "../core-primitives/time-utils" }
 
+
+[features]
```

Comment:
This shouldn't be necessary. Rather, the `evm_std` in `ita-stf` should include `evm` feature by default: 

E.g. in https://github.com/integritee-network/worker/blob/89c900c7aa5453a1faa7075d930c62c846144b81/app-libs/stf/Cargo.toml#L47-L49
Just like in https://github.com/integritee-network/worker/blob/89c900c7aa5453a1faa7075d930c62c846144b81/app-libs/sgx-runtime/Cargo.toml#L54

Reply:
it is required because of the compile flags ind ita-stf.

Reply:
Exactly, That's why ita-stf itself should import the `evm` feature.